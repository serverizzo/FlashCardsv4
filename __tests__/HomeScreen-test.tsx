import { render } from '@testing-library/react-native';

import HomeScreen, { CustomText } from '@/app/(tabs)/index';

describe('<HomeScreen />', () => {
  test('Text renders correctly on HomeScreen', () => {
    const { getByText } = render(<HomeScreen />);

    getByText('Welcome!');
  });

  test('CustomText renders correctly', () => {
    const tree = render(<CustomText>Some text</CustomText>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});