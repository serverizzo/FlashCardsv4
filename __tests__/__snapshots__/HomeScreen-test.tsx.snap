// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> CustomText renders correctly 1`] = `
<Text>
  Some text
</Text>
`;
